src/game_stats.c

```diff
@@ -0,0 +1,387 @@
+#include "global.h"
+#include "math_util.h"
+#include "game_stats.h"
+
+struct GAME_STATS {
+    u32 statsWords[NUM_GAME_STATS_WORD];
+    u16 statsHalf[77];
+    u16 dummy;
+    u16 unk_1BC;
+    u16 unk_1BE;
+};
+
+static void GameStats_Release(GAME_STATS *gameStats, int statIdx);
+static void GameStats_Acquire(GAME_STATS *gameStats, int statIdx);
+static u32 GameStats_GetValue(GAME_STATS *gameStats, int statIdx);
+static u32 GameStats_SetValue(GAME_STATS *gameStats, int statIdx, u32 value);
+static u32 GameStats_GetMaxValue(int statIdx);
+static u16 GameStats_GetStdInc(int statIdx);
+
+static u8 _0210F864[] = {
```

Comment:
data for whether the specified game stat has a high cap (TRUE) or low cap (FALSE)